backend/core/domains/bookingflow/views/booking_session_views.py

```
@@ -275,17 +289,21 @@ def update_session_data(self, request, session_uuid=None):
             step_data = request.data.get('step_data', {})
             mark_completed = request.data.get('mark_completed', False)
             
+            print(f"API DEBUG: step_id={step_id}, mark_completed={mark_completed}, session_uuid={session_uuid}")
+            
             if not step_id:
                 return Response(
                     {"detail": "step_id is required"},
                     status=status.HTTP_400_BAD_REQUEST
                 )
             
+            print(f"API DEBUG: About to call service method")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)